lib/BenchShow/Common.hs

```diff
@@ -341,6 +345,7 @@ defaultConfig = Config
     , presentation      = Groups Absolute
     , estimator         = Median
     , threshold         = 3
+    , failureField      = Nothing
```

Comment:
Instead of introducing another option for this. We can use `selectBenchmarks` to achieve this. In that filter we can select only those benchmarks that exceed the threshold. Only those benchmarks will be reported in the output. If no benchmark exceeds the threshold, no output will be generated. If we do this then we won't need the rest of the code you added below.

We can have an option (e.g. --silent) to say do not generate a report (or any output) if no benchmarks are selected. If we have that then we can say the run succeeded if there was no output produced.

Also, to make it reliable on benchmark renaming we also have to fail if the two runs being compared have a different set of benchmark names. We already report this in the output. So even in the silent mode we should keep this output so that we can fail. In such failures we have to do a manual check to see if any regression occurred.

Reply:
See #27 as a pre-requisite

Reply:
See #28